xapi-idl/cluster/cluster_interface.ml

```diff
@@ -62,7 +62,8 @@ type cluster_config = {
   ; config_version: int64
   ; cluster_token_timeout_ms: int64
   ; cluster_token_coefficient_ms: int64
-  ; pems: pems option
+  ; verify_tls_certs: bool
```

Comment:
I'm not sure whether `rpc` is smart enough to load a cluster_config that didn't have verify_tls_certs and upgrade it to 'false', maybe we need to specify an @default value of false (similarly for list a default value of empty, but I'd assume that'd already know that).
There should be some unit tests, either here or in xapi-clusterd that tests for config upgrades. See whether they pass or fail currently.

Reply:
so something like might help:
```
verify_tls_certs: bool [@default false]
pems: pems [@default []]
```

the other type modified below is only used in xapi <->xapi-clusterd communication so doesn't necessarily have to be backward compatible